src/ts/components/pages/EditProfile/EditProfile.tsx

```diff
@@ -145,7 +151,7 @@ class UnwrappedEditProfile extends React.PureComponent<EditProfileProps, EditPro
                 description: store.user.description,
                 profilePicture: undefined,
                 wallet: isProducerUser(store.user) ? store.user.wallet : "",
-                pairingLink: isProducerUser(store.user) ? store.user.pairingLink : "",
+                pairingLink: isProducerUser(store.user) ? `${process.env.REACT_APP_OBYTE_PROTOCOL}:${process.env.REACT_APP_OBYTE_PAIRING_CODE}#` + (store.user.pairingLink + "").split("#", 2)[1] : "",
```

Comment:
This seems quite hacky, are there plans on improving this in the back log?

Reply:
Possibly. The optimal solution would be to change the pairingLink all the way in the backend, but Casper has given the greenlight for a sorta hacky, string manipulation solution for now.

Reply:
I think this is good enough for now :)